packages/patternfly-4/react-core/src/helpers/PopoverBase/PopoverBase.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
```

Comment:
Since this is only one file, can we make it Typescript and make a named export `PopoverBase`?

Reply:
This is the plan, I'd like to do it in a follow-up PR if that's ok?

Reply:
This work has proved tricky for me so I think sending smaller PR's might be good for catching any issues and make the progress a little more steady.